corpus/pages/views.py

```diff
@@ -1,8 +1,15 @@
+from config.models import Society
 from django.shortcuts import render
 
-# Create your views here.
-
 
 def index(request):
-    args = {}
-    return render(request, "pages/index.html", args)
+    # Get all societies to render on landing page Societies section
+    societies = Society.objects.all()
+
+    return render(
+        request,
+        "pages/index.html",
+        {
+            "societies": societies,
+        },
```

Comment:
Would it be cleaner to define this as a different variable, and pass that variable here? In case in the future we want to add more and more parameters.

Reply:
Doesn't really matter imo, you'll be declaring the same number of lines (whether it be in a variable before this call or creating a `dict` within this call itself. Also, those lines would have to come right before the `render` call so that we finish all data processing before packaging it in a `dict` for the template.
It's a very minimal change, but I think keeping it this way makes it easier to directly see what values are being passed into the template.